woocommerce/woocommerce
# enhance data retention option for orders with status refunded

### All Submissions:






### Changes proposed in this Pull Request:
Add additional data retention options for orders with status refunded.
See Issue for a detailed description: #28800



Closes #28800 .

### How to test the changes in this Pull Request:

1. go to settings: wp-admin/admin.php?page=wc-settings&tab=account
2. set a retention period
3. check after the retention period if it got removed correctly

### Other information:




### Changelog entry

> Adds option to add a data retention period for orders with status refunded
